src/app/clusters/power-source-configuration-server/power-source-configuration-server.cpp

```diff
@@ -33,26 +34,34 @@ using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::PowerSourceConfiguration::Attributes;
+using namespace chip::app::Clusters::PowerSource;
 
 namespace {
 
+// Can be upto max endpoints count
+constexpr uint8_t kMaxPowerSources = 10;
```

Comment:
Per spec, the Sources attribute has max length 6, no?

Reply:
I had not looked at this. I will fix it.